### DISCUSSION TITLE
Sharing thumbnail other than header/teaser image?

### DISCUSSION THREAD

**Discussion by mcfisch** [mmistakes/minimal-mistakes#3159] [Q&A] [answered]
I've noticed, that when sharing WhatsApp doesn't use any pictures at all, and Twitter, Facebook and LinkedIn use the header image. But I did specify a teaser image, and that is used in the grid view of the recommendations sections of the blog.

Is there a way to make the teaser picture the preferred picture for shared links if it exists? The issue in my specific case is that the header is a different picture than the teaser, made specifically to be displayed at the page's top. The teaser picture on the other hand is specifically made as a thumbnail, but it only gets used in these blog internal recommendations. Now I'd also like that to be the share picture - if it exists. I don't want to change the behavior for other pages, though.

Thanks,
Mario

PS: 
The page in questions is https://blog.mcfisch.com/woodworking/My-first-Woodworking-Project/
resp. https://github.com/mcfisch/mcfisch.github.io/blob/main/_posts/2021-09-19-My-first-Woodworking-Project.md

**Comment by mmistakes** [comment, marked as answer]
The theme supports this, check the documentation here on how to do it.
https://mmistakes.github.io/minimal-mistakes/docs/layouts/#open-graph--twitter-card-images

**Reply by mcfisch** [reply]
Thank you, I missed that when looking this stuff up. Added the `og_image` to the post and all is working now. 👍🏻
